probot/package.json

```diff
@@ -14,7 +14,7 @@
   ],
   "scripts": {
     "dev": "nodemon",
-    "start": "probot run ./index.js",
+    "presolver": "probot run ./index.js",
```

Comment:
When I use start, lerna loads the probot index page after / over the dashboard... looking into running it with shelljs at the /probot route.